src/wymeditor/editor/native-edit-registration.js

```diff
@@ -19,20 +19,36 @@ WYMeditor.NativeEditRegistration = function (wym) {
     // https:/github.com/mightyiam/edited
     nativeEditRegistration.edited = new WYMeditor.EXTERNAL_MODULES.Edited(
         wym._doc.body,
-        function () {
-            nativeEditRegistration._onNativeEdit.call(nativeEditRegistration);
```

Comment:
BTW this line has a mistake. The call to `Function.prototype.call` is unnecessary. It could have been just `nativeEditRegistration._onNativeEdit()`.